### DISCUSSION TITLE
The endAt can be judged better

### DISCUSSION THREAD

**Discussion by xiangyeping** [prometheus/prometheus#11560] [Q&A]
### Proposal

For example, in an actual scenario, the CPU utilization of a machine jitters. The exception lasts for 10 minutes, returns to normal at the eleventh minute, and returns to abnormal at the twelfth minute. This situation should not be alarmed, and the reasonable handling method should be to reset it to recovery after a normal period of time (for example, 3 minutes). At present, many alarms will be triggered as long as the normal mode is set to recovery immediately. In addition, once the data is not collected, the recovery alarm will be triggered immediately. It is necessary to judge how many cycles before resuming the alarm

**Comment by roidelapluie** [comment]
endsAt is an internal value between Prometheus and alertmanager. I do not understand your problem exactly, can you share the recording rules?
